web/admin/src/editCell.js

```diff
@@ -53,6 +53,7 @@ export class CellEdit extends Component {
             className="ddLink"
             id={task.key}
             type="text"
+            onChange={e => console.log('edit')}
```

Comment:
remove?

Reply:
I was getting a warning for a textInput without any onChange is the reasoning here

Reply:
Hm... this warning is probably meant to steer you in a different direction.  Is this input meant to be edited?  If not, it should probably be disabled (with no onChange).